### PR TITLE
Replaced decorate imports with new decorate repo

### PR DESCRIPTION
**Pull request by EduardGomezEscandell**
We have copy-pasted the `decorate` submodule quite a few times accross ubuntu and canonical repos. A while a go we decided to keep a centralized version instead of many copies in different repos, and so [ubuntu/decorate](https://github.com/ubuntu/decorate) was born.

That repo was copy-pasted from adsys' commit [3cceb75a3de66874e2f472e595e73dcb7913869e](https://github.com/ubuntu/adsys/tree/3cceb75a3de66874e2f472e595e73dcb7913869e). There have been no modifications to either adsys' nor the new repo's code, so the files should still be identical.

This pull request:
1. Removes the internal/decorate package.
2. Replaces all `decorate` imports with `github.com/ubuntu/decorate`.

### CODE REVIEW

**Review by GabrielNagy — APPROVED** [review]
LGTM! Can we add this block to `debian/copyright`? Copy-pasted from ubuntu-proxy-manager:

```
Files: vendor/github.com/ubuntu/decorate/*
Copyright: Canonical
License: MIT
```

**Comment by EduardGomezEscandell** [comment]
Done! Please review I did it right, I've never worked with debian copyright files and I don't know if you sort them according to any patterns; I just appended it after the last `Files:` entry.

File: debian/copyright [hunk 1]
```diff
@@ -322,6 +322,10 @@ Files: vendor_rust/android_system_properties/*
 Copyright: 2013, 2016 Nicolas Silva
 License: MIT or Apache-2.0
 
+Files: vendor/github.com/ubuntu/decorate/*
+Copyright: Canonical
+License: MIT
+
```
[the remark this answers is not in the document]
Comment (EduardGomezEscandell) [reply debian/copyright]: Makes sense, I knew there had to be some ordering 😅. I ammended and force-pushed the last commit.

**Comment by EduardGomezEscandell** [comment]
Feel free to merge, I have no write access

**Pull request merged**